### PR TITLE
setup frontend environment

### PR DESCRIPTION
**Pull request by nari-z**

### CODE REVIEW

File: frontend/dist/.gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+*
```
Comment (enokawa) [inline frontend/dist/.gitignore]: I think the `.gitignore` file is only need `frontend/.gitinogre`.

File: frontend/.gitignore [hunk 2]
```diff
@@ -0,0 +1,4 @@
+# elm-package generated files
+elm-stuff
+# elm-repl generated files
+repl-temp-*
```
Comment (enokawa) [inline frontend/.gitignore]: This file is not generated in my environment.

```sh
$ ls -l
total 16
-rw-r--r--  1 enokawa  staff  258 Jul  4 20:50 README.md
drwxr-xr-x  3 enokawa  staff   96 Jul  4 20:53 dist
drwxr-xr-x  3 enokawa  staff   96 Jul  4 20:51 elm-stuff
-rw-r--r--  1 enokawa  staff  517 Jul  4 20:50 elm.json
drwxr-xr-x  3 enokawa  staff   96 Jul  4 20:50 src
```

Comment (enokawa) [reply frontend/.gitignore]: And `dist/` is not included. Don't you want it?  
I think `dist/` direcotory should not be included repo.

**Review by enokawa — CHANGES_REQUESTED** [review]

File: frontend/.gitignore (hunk 2 above)
Comment (enokawa) [reply frontend/.gitignore]: Last one. Don't need a new line at the end?  
Please enable **Insert Final Newline** settings for your editor.

**Review by enokawa — CHANGES_REQUESTED** [review]

File: frontend/dist/.gitignore (hunk 1 above)
Comment (nari-z) [reply frontend/dist/.gitignore]: OK, merged .gitignore

File: frontend/.gitignore (hunk 2 above)
Comment (nari-z) [reply frontend/.gitignore]: please try again.
`elm make ./src/Main.elm --output=dist/index.html`

**Review by enokawa — APPROVED** [review]

**Comment by enokawa** [comment]
LGTM.

**Pull request merged**